internal/infrastructure/providers/weatherapi_provider.go

```diff
@@ -3,18 +3,18 @@ package providers
 import (
 	"context"
 	"encoding/json"
-	"fmt"
 	"io"
 	"net/http"
+	"net/url"
 	"weather-forecast/internal/domain/models"
 	"weather-forecast/internal/infrastructure/apperrors"
 	"weather-forecast/internal/infrastructure/logger"
 )
 
-const LOCATION_NOT_FOUND = 1006
+const NotFoundWeatherAPI = 1006
```

Comment:
```suggestion
const NotFoundWeatherAPIErrorCode = 1006
```
І можна зробити приватною константою всередині GetWeatherByCity

Reply:
Виправив